linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/tests/stig_correct.pass.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_stig
+
```

Comment:
Please add here
# platform = Red Hat Enterprise Linux 8
cause the stig profile in rhel7 does not cotain this rule.

Reply:
@vojtapolasek good point! Updated